source/php/Modules/FrontendForm/FormAdmin.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace EventManager\Modules\FrontendForm;
+
+use WpService\Contracts\AddFilter;
+use WpService\Contracts\GetPostTypeObject;
+use WpService\Contracts\GetPostType;
+use AcfService\Contracts\GetFieldGroups;
+
+class FormAdmin
```

Comment:
Can we implement from interface?

Reply:
Resolved by splitting into multiple functions.